code/game/jobs/job/medical.dm

```diff
@@ -94,15 +94,13 @@
 							/datum/computer_file/program/chem_catalog,
 							/datum/computer_file/program/camera_monitor)
 
-	description = "The Doctor is a professional medic and surgeon dedicated to healing the sick and injured, at all costs.<br>\
-	A broad range of medical procedures fall under your purview - diagnostics, general treatment, surgery, and virology.<br>\
-	You are not expected to be an expert in all: specializing in an area is fine. Divide tasks amongst colleagues, with CBO guidance.<br>\
-	Remember that chemistry has a dedicated specialist. Avoid this department unless it is notably short-staffed.<br>\
-	Due to the nature of your work, you may find yourself confined to the department for the shift majority. Don't abandon patients."
+	description = "The Doctor is a professional medic and surgeon dedicated to healing the sick and injured at all costs.<br>\
+	A broad range of medical procedures fall under your purview - diagnostics, general treatment, surgery and virology.<br>\
+	You are not expected to be an expert in all: specializing in an area is fine. Divide tasks amongst colleagues with CBO guidance.<br>\
+	Due to the nature of your work you may find yourself confined to the department for the shift majority. Don't abandon patients."
 
 	duties = "Heal the sick and injured, whatever their complaint.<br>\
-		Diagnose illnesses, offer general services, perform surgery, or even study viruses.<br>\
-		Fill in at chemistry if a Chemist is unavailable."
+		Diagnose illnesses, offer general services, perform surgery, or even study viruses.<br>\"
```

Comment:
Checks are failing because of the way you've edited the blurbs. You need to remove this <br>\. Anything that follows a \ string is ignored until the next line. This is normally used to organize text to fit on screen without having to scroll as a code cleanliness thing.

There might be others but this is the one that stood out. I didn't review all of this.

Reply:
Ahh so it shouldn’t end on that, alright